docs/index.md

```diff
@@ -16,6 +16,8 @@ The YugabyteDB Aeon Provider can be used to interact with the resources provided
 VPC Peerings, Read Replicas and so on. The provider needs to be configured with appropriate credentials before it can base used. The navigation bar on the left
```

Comment:
Slight rewording for this, can use in both files:

YugabyteDB is a high-performance, cloud-native, PostgreSQL-compatible distributed SQL database. It is best fit for cloud-native OLTP (real-time, business-critical) applications that need absolute data correctness and at least one of the following: scalability, high tolerance to failures, or globally-distributed deployments. YugabyteDB Aeon is a fully managed YugabyteDB-as-a-Service, providing all the features of YugabyteDB without the operational overhead.

Use the YugabyteDB Aeon Provider to interact with the resources provided by YugabyteDB Aeon, including YugabyteDB clusters, allow lists, VPCs, VPC peerings, read replicas, and more. You must configure the provider with appropriate credentials before using it. Use the left-side navigation to access the resources supported by the provider and related guides.

Note: Don't use `-pre` releases for production.